feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/captive.c

```diff
@@ -267,6 +267,10 @@ void vif_state_captive_portal_options_get(struct schema_Wifi_VIF_State *vstate)
 					set_captive_portal_state(vstate, &index,
 							captive_portal_options_table[i],
 							buf);
+				} else if (!strcmp(buf,"radius")) {
```

Comment:
All 3 if ... else if ... do the same thing.  Therefore collapse into 1 statement.

Reply:
Done, thank you!